dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/DefaultIdentifiableObjectManager.java

```diff
@@ -280,22 +272,25 @@ public <T extends IdentifiableObject> T get( Class<T> type, String uid )
         return store.getByUid( uid );
     }
 
+    @Nonnull
     @Override
-    public <T extends IdentifiableObject> T load( Class<T> type, String uid )
+    public <T extends IdentifiableObject> T load( @Nonnull Class<T> type, @Nonnull String uid )
         throws IllegalQueryException
     {
         IdentifiableObjectStore<T> store = getIdentifiableObjectStore( type );
 
         if ( store == null )
         {
-            return null;
+            throw new IllegalQueryException( "No store registered for objects of type: " + type );
```

Comment:
OBS! Either this method should return an object or throw an exception. This makes it consistent.